### ISSUE TITLE
Cursors aren't displayed when axis scale is set to 'time'

### ISSUE THREAD

**Issue by hrmoller** [FormidableLabs/victory#888]
# Bugs and Questions

### Checklist

* [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

* [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

* [x] I am using the latest version of Victory

* [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

Cursors are not visible when using `VictoryCursorContainer` with a `VictoryChart` that contains a `<VictoryAxis scale={{x: 'time'}} />`

Changing the scale to any other allowed value (`linear`, `log`, `sqrt`) causes the cursors to be displayed as expected.

### Reproduction
[https://codesandbox.io/s/3yyn3mk9m6](https://codesandbox.io/s/3yyn3mk9m6)

**Comment by jbockerstette** [comment]
This was fixed by pr #543 which has been pulled in. Maybe you will need to wait for next release or you can monkey patch it. You need to monkey patch BrushHelpers.withinBounds.

**Comment by hrmoller** [comment]
Alright, thank you for letting me know.

Just to anyone coming by this with the same issue I ended up doing it this way where the scale prop is removed:

```
<VictoryAxis
  offsetY={40}
  style={{
    axis: {stroke: axisColor},
    grid: {stroke: gridColor, strokeWidth: strokeWidthGrid},
    ticks: {stroke: gridColor, strokeWidth: 1, size: 10}
  }}
  tickCount={24}
  tickValues={tickValues}
  tickFormat={t => moment(t).format('LT')}
  tickLabelComponent={<VictoryLabel angle={35} />}
/>
```
